ui/raidboss/data/06-ew/raid/p9n.txt

```diff
@@ -1,6 +1,364 @@
+296.7 "Dualspell" sync / 1[56]:[^:]*:Kokytos:811D:/
+300.0 "--sync--" sync / 1[56]:[^:]*:Kokytos:8123:/ # ice marker
+304.4 "Fire III (Static)" sync / 1[56]:[^:]*:Kokytos:811E:/
+304.4 "Blizzard III (Shrinking)" sync / 1[56]:[^:]*:Kokytos:8121:/
+304.8 "--sync--" sync / 1[56]:[^:]*:Kokytos:811C:/
+
+313.9 "Global Spell" sync / 1[56]:[^:]*:Kokytos:8141:/ jump 500.0
+317.1 "Disgorge" #sync / 1[56]:[^:]*:Kokytos:8124:/
+329.3 "Ravening (Fighter)" #sync / 1[56]:[^:]*:Kokytos:8119:/
+350.7 "Archaic Rockbreaker" #sync / 1[56]:[^:]*:Kokytos:8128:/
+
+# -> after Dualspell
+500.0 "Global Spell" sync / 1[56]:[^:]*:Kokytos:8141:/
+500.9 "--sync--" sync / 1[56]:[^:]*:Kokytos:8142:/
+503.2 "Disgorge" sync / 1[56]:[^:]*:Kokytos:8124:/
+507.2 "--sync--" sync / 1[56]:[^:]*:Kokytos:8115:/
+510.3 "--sync--" sync / 1[56]:[^:]*:Kokytos:8115:/
+
+
+
+# Fighter phase 1
+515.4 "Ravening (Fighter)" sync / 1[56]:[^:]*:Kokytos:8119:/
+524.5 "--sync--" sync / 1[56]:[^:]*:Kokytos:8127:/
+527.6 "--sync--" sync / 1[56]:[^:]*:Kokytos:8127:/
+530.7 "--sync--" sync / 1[56]:[^:]*:Kokytos:8127:/
+536.8 "Archaic Rockbreaker" sync / 1[56]:[^:]*:Kokytos:8128:/
+538.4 "Shockwave" sync / 1[56]:[^:]*:Kokytos:8129:/
+541.1 "--sync--" sync / 1[56]:[^:]*:Kokytos:8127:/
+544.2 "--sync--" sync / 1[56]:[^:]*:Kokytos:8127:/
+545.3 "Archaic Rockbreaker 1" sync / 1[56]:[^:]*:Kokytos:812A:/
+547.3 "--sync--" sync / 1[56]:[^:]*:Kokytos:8127:/
+550.3 "Archaic Rockbreaker 2" sync / 1[56]:[^:]*:Kokytos:812A:/
+556.9 "Archaic Demolish" sync / 1[56]:[^:]*:Kokytos:812F:/
+557.9 "--sync--" sync / 1[56]:[^:]*:Kokytos:8130:/
+564.0 "Gluttony's Augur" sync / 1[56]:[^:]*:Kokytos:8116:/
+564.4 "Gluttony's Augur" sync / 1[56]:[^:]*:Kokytos:8117:/
+571.1 "Ascendant Fist" sync / 1[56]:[^:]*:Kokytos:8131:/
+573.2 "--sync--" sync / 1[56]:[^:]*:Kokytos:8127:/
+576.3 "--sync--" sync / 1[56]:[^:]*:Kokytos:8127:/
+583.4 "Front Combination/Rear Combination" sync / 1[56]:[^:]*:Kokytos:814[8A]:/
+583.6 "Outside Roundhouse" sync / 1[56]:[^:]*:Kokytos:812C:/
+586.6 "Swinging Kick" sync / 1[56]:[^:]*:Kokytos:812[DE]:/
+590.7 "Disgorge" sync / 1[56]:[^:]*:Kokytos:8125:/
+594.7 "--sync--" sync / 1[56]:[^:]*:Kokytos:8115:/
+597.8 "--sync--" sync / 1[56]:[^:]*:Kokytos:8115:/
+599.0 "--sync--" sync / 1[56]:[^:]*:Kokytos:8144:/
+
+
+
+# Behemoth phase 1
+603.7 "Ravening (Behemoth)" sync / 1[56]:[^:]*:Kokytos:811A:/
+621.0 "Charybdis" sync / 1[56]:[^:]*:Kokytos:8132:/
+622.0 "--sync--" sync / 1[56]:[^:]*:Kokytos:8133:/
+627.1 "Beastly Roar" sync / 1[56]:[^:]*:Kokytos:8138:/
+629.2 "Beastly Bile" sync / 1[56]:[^:]*:Kokytos:813D:/
+631.2 "--sync--" sync / 1[56]:[^:]*:Kokytos:813E:/
+638.3 "Pulverizing Pounce" sync / 1[56]:[^:]*:Kokytos:813F:/
+640.4 "--sync--" sync / 1[56]:[^:]*:Kokytos:8140:/
+643.5 "--sync--" sync / 1[56]:[^:]*:Kokytos:8140:/
+653.6 "Gluttonous Rampage" sync / 1[56]:[^:]*:Kokytos:8139:/
+653.8 "Ravenous Bite 1" sync / 1[56]:[^:]*:Kokytos:813A:/
+655.9 "Ravenous Bite 2" sync / 1[56]:[^:]*:Kokytos:813A:/
```

Comment:
These are tankbusters on both the tanks, one after another, and also has distance tethers on them, which should probably have a trigger but I'm not sure what trigger would be best, if there is something similar pre-existing.

Reply:
@quisquous will have to weigh in for a final decision on this, but it's probably worth doing something like "buster on YOU -- stretch tethers". It's longer than any of us would prefer, but we do want to convey all the information.

Reply:
Yeah, there's nothing preexisting here; I think this is the first proximity buster.  I think `Buster on YOU -- stretch tethers` is kind of wordy, but I'm not sure what else to say here either.

Reply:
Added as "Tankbuster on ${player}" / "Buster on YOU -- stretch tethers".

The ability is unfortunately not actually targeted on the tanks, as far as I can see it's just the tether line and the head marker, so I based it off the head marker. It does have the slight downside that the people who get the "tankbuster on ${player}" trigger will get two of them, about 2 seconds apart, since the tethers and markers are about two seconds apart.